torch_geometric/typing.py

```diff
@@ -20,6 +22,9 @@
 
 Metadata = Tuple[List[NodeType], List[EdgeType]]
 
+# A representation of a feature tensor
+FeatureTensorType = Union[torch.TensorType, numpy.ndarray]
```

Comment:
```suggestion
TensorType = Union[torch.Tensor, numpy.ndarray]
```

Reply:
I changed this from `TensorType` to `FeatureTensorType` to contrast with the future `EdgeTensorType`. Do you prefer a different naming?